setup.py

```diff
@@ -342,6 +342,13 @@ def report(*args):
 cmake = CMake()
 
 
+def is_source_hipified():
+    with open("torch/csrc/utils/cuda_enabled.h") as f:
```

Comment:
This will make it work even if invoked as `python pytorch/setup.py develop` or something like that
```suggestion
    with open(os.path.join(os.path.dirname(__file__), "torch/csrc/utils/cuda_enabled.h")) as f:
```

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/114309/commits/9f028458523aefc1dc09612eff095e2ea416921f